app/views/school/devices/cannot_order.html.erb

```diff
@@ -21,5 +21,15 @@
     <p class="govuk-body">You can still <%= govuk_link_to 'request devices for disadvantaged children', school_request_devices_path %> from any year group who:</p>
 
     <%= render partial: 'shared/devices/who_to_request_for_list' %>
+
+    <% unless @user.capable_of_placing_orders? %>
```

Comment:
This isn't quite right, this message should only show to users who will never have a TechSource account, not for those who will have an account that just isn't set up yet. ie When they were invited they were not given permission to order.

Reply:
For users who have a pending account, we need to show this design:
https://ghwt-design-history.herokuapp.com/ordering-devices-school/#school-can-order-but-your-techsource-account-is-not-ready-yet

Reply:
done, I wasn't aware of the other design